includes/Experiments.php

```diff
@@ -208,6 +208,17 @@ public function get_experiment_groups() {
 	 */
 	public function get_experiments() {
 		return [
+			/**
+			 * Author: @littlemilkstudio
+			 * Issue: 6708
+			 * Creation date: 2021-03-23
+			 */
+			[
+				'name'        => 'enableHiddenStickers',
+				'label'       => __( 'Enable Hidden Stickers', 'web-stories' ),
```

Comment:
Perhaps strike the `hidden` from here so we can re-use that same flag later for the real thing. :-)